perma_web/Dockerfile

```diff
@@ -69,14 +68,14 @@ COPY perma_web/Pipfile /perma/perma_web
 COPY perma_web/Pipfile.lock /perma/perma_web
 RUN pip3 install -U pip \
     && pip install pipenv \
-    && pipenv --python 3.7 install --ignore-pipfile --dev \
+    && pipenv --python 3.9 install --ignore-pipfile --dev \
     && rm Pipfile.lock
 
 # Install Chrome
 # technique from https://github.com/SeleniumHQ/docker-selenium/blob/master/NodeChrome/Dockerfile.txt
 ARG chrome-layer-cache-buster
 RUN apt-get --allow-releaseinfo-change update \
-    && apt-get install -y unzip libappindicator3-1 libasound2 libatk-bridge2.0-0 libatk1.0-0 libatspi2.0-0 libgbm1 libgtk-3-0 libnspr4 libnss3 libxcomposite1 libxcursor1 libxi6 libxrandr2 libxss1 libxtst6 xdg-utils \
```

Comment:
`libappindicator3-1` is not in bullseye and does not appear in the list of deps from `docker-compose exec web dpkg -s google-chrome-stable`. I haven't confirmed that the packages installed here match those deps one-to-one, but the installer would bark if we were missing anything.